pulsar-flink/src/main/java/org/apache/flink/streaming/connectors/pulsar/serde/JsonRowDeserializationSchema.java

```diff
@@ -38,6 +38,21 @@
  */
 public class JsonRowDeserializationSchema implements DeserializationSchema<Row> {
 
+    /*
+        What to do when detecting that a json line cannot be deserialized :
+        (1).false : Throw A IOException and Terminate application。
+        (2).true  : Ignore the error line and add a null line。
+     */
+    private Boolean ignoreJsonFormatError = false;
```

Comment:
nit: I would suggest using `boolean` rather than `Boolean` to avoid boxing and unboxing.

Reply:
it is a good idea